Lombiq.Tests.UI.Samples/Tests/TenantTests.cs

```diff
@@ -32,14 +29,20 @@ public TenantTests(ITestOutputHelper testOutputHelper)
             async context =>
             {
                 const string tenantAdminName = "tenantAdmin";
+
                 await context.SignInDirectlyAsync();
 
-                // Create the tenant with custom admin name.
-                await context.CreateAndEnterTenantAsync(
-                    TestTenantDisplayName,
+                // Create the tenant with a custom admin user.
+                await context.CreateAndSwitchToTenantAsync(
+                    TestTenantUrlPrefix,
```

Comment:
The display name isn't replaced, it's set on line 41, and the method requires a technical name and a prefix (as this is what's needed for creating the tenant itself, the rest is setup). The duplication here is only an artifact of simplification in the test (since "test" as the technical name and URL prefix is both suitable), but I'll introduce a second const for it to be nicer.